ifrs17-template/Test/Data/IfrsVariableBenchmarks/BM_CH_2021_3__C_Projection_DTP.csv

```diff
@@ -0,0 +1,12 @@
+@@Main,,,,,,,,,,,
+Month,ReportingNode,Scenario,Year,,,,,,,,
+3,CH,,2021,,,,,,,,
+@@C,,,,,,,,,,,
+AccidentYear,AmountType,AocType,DataNode,EconomicBasis,EstimateType,Novelty,Values0,Values1,Values2,Values3,Values4
```

Comment:
EA and IA steps are missing. 
EA should exists only for the current period.

Reply:
Locked in is Zero